fluent-package/apt/systemd-test/update-to-next-version-service-status.sh

```diff
@@ -42,10 +42,13 @@ else
     (! systemctl is-enabled fluentd)
 fi
 
-if [ "$status_before_update" = active ] || [ "$enabled_before_update" = enabled ] ; then
-    # The service should restart automatically after update
+if [ "$status_before_update" = active ]; then
+    # The service should NOT restart automatically after update
+    systemctl is-active fluentd
+    test $main_pid -eq $(systemctl show --value --property=MainPID fluentd)
+elif [ "$enabled_before_update" = enabled ] && [ "$status_before_update" = inactive ]; then
```

Comment:
Isn't the latter condition unnecessary?

```suggestion
elif [ "$enabled_before_update" = enabled ]; then
```

Reply:
Indeed. it is unnecessary.